tests/structured-matching-service.test.ts

```
@@ -6,6 +6,7 @@ const { mockGenerateText, mockWithRetry } = vi.hoisted(() => ({
 }));
 
 vi.mock("../src/lib/ai-models", () => ({
+  embeddingModel: "mock-embedding-model",
```

Biome (2.4.10)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)